src/OT/Color/COLR/COLR.hh

```diff
@@ -97,10 +97,16 @@ public:
 
     if (color_index != 0xffff)
     {
-      unsigned int clen = 1;
-      hb_face_t *face = hb_font_get_face (font);
+      if (palette != HB_PAINT_PALETTE_INDEX_CUSTOM)
```

Comment:
palette_index here as well?

Reply:
Confusingly this file calls it palette. I'll rename.